justfile

```diff
@@ -145,24 +145,55 @@ build-cni-plugin-test-image *args='--load':
         --tag={{ _cni-plugin-test-image }} \
         {{ args }}
 
-# Build and load images for cni-plugin
-cni-plugin-test-integration-deps: build-cni-plugin-image build-cni-plugin-test-image _k3d-ready
-    @just-k3d import {{ _cni-plugin-test-image }} {{ cni-plugin-image }}
+##
+## CNI plugin integration
+## 
 
-# Run cni-plugin integration tests after preparing dependencies
-# For new scenarios, add them after cni-plugin-test-integration-deps
-cni-plugin-test-integration: cni-plugin-test-integration-deps cni-plugin-test-integration-flannel
 
-# Run flannel integration tests without preparing dependencies
-cni-plugin-test-integration-flannel:
-    SCENARIO=flannel TEST_CTX="k3d-$(just-k3d --evaluate K3D_CLUSTER_NAME)" ./cni-plugin/integration/run.sh
+# Run cni-plugin integration tests after preparing dependencies By default,
+# runs "flannel" scenario, behavior can be overridden through
+# `cni-integration-scenario` variable
```

Comment:
I couldn't find where this variable is used.

Reply:
Yeah I removed it and forgot to update the docs. It was replaced by an env var.